elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -10,9 +10,12 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
     BorderRight: 0,
     BorderBottom: 0,
     BorderLeft: 0,
-  ));
+  )
+);
+
+@include pfe-local-debug;
```

Comment:
🧗‍♀️ These debugging statements should be removed. 🤔 I wonder if there's a way to automatically strip them from a PR on commit with a husky script.

Reply:
What do you think about using a variable to render them useless all at once? like if the mixin had this flag:

```scss
@mixin pfe-local-debug($map: $LOCAL-VARIABLES) {
  @if $debug == true {
    :root {
        @include pfe-print-local($map);
    }
  }
} 
```

then the debug statements could still appear in the component files but not do anything unless you toggle that variable on?

Reply:
I feel like it's kind of like leaving a console.log in a JS file.  It just feels a bit messy.